From 7e53b6bd19012102e1c06d9a12fdac64948e6a93 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Wed, 14 Nov 2018 09:18:51 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399pro-npu: fix up the pd_pcie qos
 node

add usb2 and usb3 qos node,
set qos_pcie\qos_usb2\qos_usb3 status = "disabled" by default.

Change-Id: I20653cc485c4c923b3b89af8e801634f4d7447a3
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi | 17 ++++++++++++++++-
 1 file changed, 16 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi b/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
index 7cd3f50238a0..c8cc49a269df 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
@@ -215,6 +215,19 @@
 	qos_pcie: qos@fe880000 {
 		compatible = "syscon";
 		reg = <0x0 0xfe880000 0x0 0x20>;
+		status = "disabled";
+	};
+
+	qos_usb2: qos@fe890000 {
+		compatible = "syscon";
+		reg = <0x0 0xfe890000 0x0 0x20>;
+		status = "disabled";
+	};
+
+	qos_usb3: qos@fe890080 {
+		compatible = "syscon";
+		reg = <0x0 0xfe890080 0x0 0x20>;
+		status = "disabled";
 	};
 
 	qos_isp: qos@fe8a0000 {
@@ -363,7 +376,9 @@
 					 <&cru HCLK_HOST_ARB>,
 					 <&cru SCLK_USB3_OTG0_REF>,
 					 <&cru SCLK_USB3_OTG0_SUSPEND>;
-				pm_qos = <&qos_pcie>;
+				pm_qos = <&qos_pcie>,
+					 <&qos_usb2>,
+					 <&qos_usb3>;
 			};
 			pd_vpu@RK1808_PD_VPU {
 				reg = <RK1808_PD_VPU>;
-- 
2.35.3

